api/management/api.go

```diff
@@ -212,7 +212,7 @@ type RealmCustomConfiguration struct {
 	OnboardingClientID                  *string   `json:"onboarding_client_id"`
 	SelfRegisterGroupNames              *[]string `json:"self_register_group_names"`
 	BarcodeType                         *string   `json:"barcode_type"`
-	AllowedBackURL                      *string   `json:"allowed_back_url"`
+	AllowedBackURLs                     []string  `json:"allowed_back_urls"`
```

Comment:
Do we want to support only new version? Backoffice will be ready in the same time?

Reply:
Selfservice and Backoffice have been updated